anvio/__init__.py

```diff
@@ -2779,10 +2779,14 @@ def TABULATE(table, header, numalign="right", max_width=0):
             ['-G', '--groups-txt'],
             {'default': None,
             'metavar': 'TEXT_FILE',
-            'help': "A 2-column tab-delimited text file specifying which group each sample belongs to. "
-                    "The first column should have the header 'sample' and contain sample names matching "
-                    "to those are in the other input data. The second column should have the header 'group' "
-                    "and contain the group name for each sample (each sample should be in 1 group only)"}
+            'help': "A 2-column tab-delimited text file specifying which group each item belongs to. "
+                    "Depending on the context, items here may be individual samples or genomes. "
+                    "The first column should have the header 'name' and contain item names matching "
```

Comment:
As I just mentioned in my previous comment, right now we require the first column to have the header `sample`, so this help text will have to change depending on whether we decide to keep the 'sample' requirement or allow any header for the first column.